Documentation/docs/learn/faq.md

```diff
@@ -92,8 +92,7 @@ formats:
 
```

Comment:
First line of the commit message needs to have one of the prefixes from ghostflow message. E.g.: `DOC: Incorrect rendering table in FAQ`. The rest of the commit message looks good.

Reply:
Yes I saw it, I changed the comment on GitHub.
No idea whether or not I have to change something in the repository as well (and even less knowledge of how to do it, probably something with `git amend` but totally no experience with it).

Reply:
I use TortoiseGit, so I am unfamiliar with git's command line syntax. I can amend to change commit message and push to your branch (if you have enabled "edits by maintainers").

Reply:
Looks like the "Allow edits by maintainers" is already enabled.